mpas_analysis/ocean/climatology_map_eke.py

```diff
@@ -0,0 +1,299 @@
+# Copyright (c) 2018,  Los Alamos National Security, LLC (LANS)
```

Comment:
@kevinrosa one more thing to fix.  Could you copy the new license header from another file.  That changed since you branched off of `develop`.

Reply:
Thanks, done!